google-ads-examples/src/main/java/com/google/ads/googleads/examples/accountmanagement/CreateCustomer.java

```diff
@@ -83,32 +83,33 @@ public static void main(String[] args) {
     }
   }
 
-  private void runExample(GoogleAdsClient googleAdsClient, Long managerId) {
+  public void runExample(GoogleAdsClient googleAdsClient, CreateCustomerParams params) {
```

Comment:
Back when we added command line params to AdWords API examples, we decided *against* passing the param object into `runExample`, since this would make it hard for users to copy/paste from examples without also copying over the param class.

Reply:
We can add an overload instead if that's better.

I'm not really convinced that this is necessary though - folks will need to rename the method anyway. I don't think changing the signature is asking much.